configcat/README.md

```diff
@@ -1,9 +1,14 @@
 # ConfigCat
 
+## About
+
+Manage features and change your software configuration using [ConfigCat feature flags][2], without the need to re-deploy code. A [10 minute trainable Dashboard][5] allows even non-technical team members to manage features directly. Deploy anytime, release when confident. Target a specific group of users first with new ideas. Supports A/B/n testing and soft launching. Provides [open-source SDKs][11] for easy integration with any web, mobile or backend application.
```

Comment:
Can we move this under the `## Overview section`?  The README follows a specific header format, `ABOUT` is not a valid header supported.